### PR TITLE
:zap: Add Semantic Multi-modal Image Synthesis paper summary

### PR DESCRIPTION
**Pull request by praeclarumjj3**

### CODE REVIEW

File: summaries/Semantically_multi-modal_image_synthesis.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+- A major modification of GroupDNet is the replacement of typical convolutions to group convolutions to achieve class-specific controllability.
+
+- GroupDNet contains one encoder and one decoder: 
+    - **Encoder**: Let *M<sub>c</sub>* denote the binary mask for class c, *X ∈ R<sup>H×W</sup>* be the input image, then first the operation: *X<sub>c</sub> = M<sub>c</sub> . X*, performs feature disentanglement. The input to the encoder is the concatenation of all the *X<sub>c</sub>* produced. All convolutions inside the encoder have C groups. The encoded Z is comprised of the class-specific Z<sub>c</sub> of all classes which serve as the controllers for their corresponding class *c* in the decoding phase.
+    - **Decoder**: Following the general idea of using all group convolutions in the generator,the convolutional layers in SPADE generator module are replaced with group convolutions resulting in the new module **Conditional Group Normalization (CG-Norm)**. A Conditional Group Block(CG-Block) is made by dynamically merging CG-Norm and group convolutions.
+
+<img src='../images/GroupDNet.png'>
+
+## Main contributions
```
Comment (dakshitagrawal97) [inline summaries/Semantically_multi-modal_image_synthesis.md:38]: Main Contributions

All the headings should have the first letter of the words capitalized, except for words such as 'a', 'an', 'with', etc.

File: summaries/Semantically_multi-modal_image_synthesis.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Semantically multi-modal image synthesis
+
+Zhen Zhu, Zhiliang Xu, Ansheng You, Xiang Bai, **CVPR 2020**
+
+## Summary
+
+The paper focuses on *semantically multi-modal image synthesis(SMIS)* task, namely, generating multi-modal images at the semantic level. It proposes a novel network for semantically multi-modal synthesis task, called **GroupDNet(Group Decreasing Network)**. The network unconventionally adopts all group convolutions and modifies the group numbers of the convolutions to decrease in the decoder, considerably improving the training efficiency over other possible solutions like multiple generators.
```
Comment (dakshitagrawal97) [inline summaries/Semantically_multi-modal_image_synthesis.md:7]: space between GroupDNet and paranthesis

GroupDNet (Group Decreasing Network)

File: summaries/Semantically_multi-modal_image_synthesis.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+**Notation**
+
+- `M`: semantic segmentation mask
+- `C`: number of semantic classes in the dataset
+- `H`: image height
+- `W`: image width
+- `G`: generator
+- `Z`: latent code
+
+For conducting label-to-image translation,G requires M as conditional input to generate images. However, in order to support multi-modal generation, another input source to control generation diversity. Normally, an encoder is applied to extract Z as the controller. Upon receiving these two inputs, the image output O can be yielded through O = G(Z, M ). However, in the SMIS task, we aim to produce semantically diverse images by perturbing the class-specific latent code which independently controls the diversity of its corresponding class.
```
Comment (dakshitagrawal97) [inline summaries/Semantically_multi-modal_image_synthesis.md:22]: wherever you use the notation, encase it within ``.

Like G should be `G`

File: summaries/Semantically_multi-modal_image_synthesis.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    - **Decoder**: Following the general idea of using all group convolutions in the generator,the convolutional layers in SPADE generator module are replaced with group convolutions resulting in the new module **Conditional Group Normalization (CG-Norm)**. A Conditional Group Block(CG-Block) is made by dynamically merging CG-Norm and group convolutions.
+
+<img src='../images/GroupDNet.png'>
+
+## Main contributions
+
+- **Appearance mixture**: We can gather the distinct styles of a person’s different body parts. Every combination of these styles presents a distinct person image, given a human parsing mask. In this way, we can create thousands of diverse and realistic person images given a person image gallery.
+
+- **Semantic manipulation**: For eg. inserting a bed in the room or replace the building with trees.
```
Comment (dakshitagrawal97) [inline summaries/Semantically_multi-modal_image_synthesis.md:42]: For e.g., inserting

Comment (praeclarumjj3) [reply summaries/Semantically_multi-modal_image_synthesis.md]: @dakshitagrawal97 Thanks for pointing out. I have corrected the mistakes. Please review again :)

**Pull request merged**